floss/utils.py

```diff
@@ -42,6 +43,48 @@ def __call__(self, parser, namespace, values, option_string=None):
         setattr(namespace, self.dest, items)
 
 
+class InstallContextMenu(argparse.Action):
+    def __init__(self, option_strings, dest, nargs=None, **kwargs):
+        super(InstallContextMenu, self).__init__(option_strings, dest, nargs=0, **kwargs)
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        # Theoretically, we don't need to check the platform again here,
+        # because non-Windows platforms will not accept the --install-right-click-menu parameter at all.
+        # This judgment is just to make the mypy type check pass.
+        # The same logic applies to `UninstallContextMenu` below.
+        if sys.platform == "win32":
+            import winreg as reg
+
+            menu_name = "Open with FLOSS"
+            menu_command = f'C:\\windows\\system32\\cmd.exe /K "python -m floss ^"%1^""'
```

Comment:
this won't work if we run the standalone binary
can you think of an implementation that supports both usage/installation methods?